kebechet/managers/initialization/initialization.py

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env python3
+# Kebechet
+# Copyright(C) 2019 Ronan Souza
+#
+# This program is free software: you can redistribute it and / or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+import logging
+import os
+import typing
+
+import git
+import yaml
+
+from kebechet.managers.manager import ManagerBase
+from kebechet.utils import cloned_repo
+
+_LOGGER = logging.getLogger(__name__)
+
+_KEBECHET_CONFIG_FILE_NAME = "kebechet.yml"
+_GIT_TOKEN_VARIABLE = "${GIT_SECRET_TOKEN}"
+_GIT_BRANCH_NAME = "kebechet-initialization"
+_GIT_COMMIT_MESSAGE = "Creating kebechet YAML file"
+_GIT_MR_LABELS = ["enhancement"]
+_GIT_MR_BODY = """" 
+                Creates a YAML file with kebechet settings following this struct:
+                
+               """
+
+
+def create_config_file(service_type, slug: str):
+    if slug == "":
+        _LOGGER.error("Initialization failed due to an error extracting repository slug")
+        return
+
+    config = {
+        "repositories": [{
+            "slug": slug,
+            "token": _GIT_TOKEN_VARIABLE,
+            "service_type": service_type,
+            "managers": [
+                {"name": "info"}
+            ]
+        }]
+    }
+
+    with open("kebechet.yml", "w") as config_file:
```

Comment:
@fridex do we like `.yml` or like even more `.yaml` ? ;)

Reply:
It should be `.yaml`, the `.yml` usually have a special meaning.